src/MIDebugEngine/Engine.Impl/Variables.cs

```diff
@@ -493,11 +494,11 @@ public string EvalDependentExpression(string expr)
             return val;
         }
 
-        internal async Task Eval(enum_EVALFLAGS dwFlags = 0, DAPEvalFlags dwDAPFlags = 0)
+        internal async Task Eval(enum_EVALFLAGS dwFlags = 0, DAPEvalFlags dwDAPFlags = 0, uint radix = 0)
```

Comment:
A few suggestion:
1. We probably should make radix required
2. We should pass it from AsyncEval as well
3. If we pass zero, I am thinking instead of querying the UI we should just assume that it is already correct.